packages/suite/src/views/suite/device-unknown/index.tsx

```diff
@@ -1,7 +0,0 @@
-// TODO: remove whole file, replaced by @suite-components/PrerequisitesGuide/components/DeviceUnknown
```

Comment:
ahh, I see, there is even a comment saying that

Reply:
Yeah 3 years ago 😄